synapse/config/auth.py

```diff
@@ -37,7 +39,9 @@ def read_config(self, config, **kwargs):
 
         # User-interactive authentication
         ui_auth = config.get("ui_auth") or {}
-        self.ui_auth_session_timeout = ui_auth.get("session_timeout", 0)
+        self.ui_auth_session_timeout = con_fig.parse_duration(
```

Comment:
Since `AuthConfig` is a subclass of `Config` this can be called as `self.parse_duration`.

It looks to me like `parse_duration` will do the correct thing if the old version is given (e.g. a raw `6000` should still work out to 6000).

We also need to update the documentation below about this configuration variable.

Reply:
@clokep I have updated PR using `self.parse_duration`.  Yes, `parse_duartion` will do the correct thing if the old version is given. I have also updated the documentation below